### PR TITLE
DOCSP-39158 1.43.0 release notes

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

Compass 1.43.0 release notes

## STAGING

https://preview-mongodbltranmdb2.gatsbyjs.io/compass/DOCSP-39158/release-notes/

## JIRA

https://jira.mongodb.org/browse/DOCSP-39158

## BUILD LOG

Latest build: https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=6633dd0db67dbf804a65cc21

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/release-notes.txt [hunk 1]
```diff
@@ -10,6 +10,33 @@ Release Notes
...
+----------------
+
+*Released May 02, 2024*
+
+New Features
+
+- Allows users to specify hints (:issue:`COMPASS-7829`)
+- Shows an error message when connecting to Stream Processing (:issue:`COMPASS-7809`)
+- Warns users when generated aggregation contains a write operation (:issue:`COMPASS-7298`)
+- Adds support to notify users for an update on linux/msi (:issue:`COMPASS-7686`)
```
Comment (jocelyn-mendez1) [inline source/release-notes.txt:23]:
```suggestion
- Adds support to notify users for an update on Linux/MSI(:issue:`COMPASS-7686`)
```

**Review by jocelyn-mendez1 — APPROVED** [review]
Thanks @ltran-mdb2, this LGTM % a small suggestion!

File: source/release-notes.txt (hunk 1 above)
Comment (ltran-mdb2) [reply source/release-notes.txt]: Done!

**Comment by ltran-mdb2** [comment]
LGTM from Maurizio Casmirri on slack

**Review by addaleax — APPROVED** [review]

**Pull request merged**